test/latin-var-name-only.test.js

```diff
@@ -0,0 +1,74 @@
+const latinVarNameOnly = require('../src/rules/latin-var-name-only-rule/index');
```

Comment:
You can avoid `/index` as CommonJS module system will automatically search for an index file in the folder. Speaking simply you can keep the original path `'../src/rules/latin-var-name-only-rule'` without changes